app/graphql/types/card_type.rb

```diff
@@ -0,0 +1,10 @@
+module Types
+  class CardType < Types::BaseObject
+    description "A card"
+    field :id, ID, null: false
+    field :column_id, ID, null: false
+    field :owner_id, ID, null: false
```

Comment:
Let's return `UserType` instead of `ID`
```suggestion
    field :owner, UserType, null: false
```

Reply:
👍